apps/epic-react/src/components/lesson-completion-toggle.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import {motion} from 'framer-motion'
+import {useReward} from 'react-rewards'
+import {useLesson} from '@skillrecordings/skill-lesson/hooks/use-lesson'
+import {useRouter} from 'next/router'
+import {trpcSkillLessons} from '@skillrecordings/skill-lesson/utils/trpc-skill-lessons'
+import toast from 'react-hot-toast'
+
+const LessonCompleteToggle = () => {
+  const {module} = useLesson()
+  const router = useRouter()
+  const {reward, isAnimating} = useReward('rewardId', 'confetti')
```

Comment:
react-rewards is hooks-based now which is much easier than the epic-react-gatsby set up